test/interpreter/operator-division.spec.ts

```diff
@@ -57,16 +57,17 @@ describe('Operator DIVISION', () => {
     expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NAME, ErrorMessage.FunctionName('FOOBAR')))
   })
 
-  // Inconsistency with Product 1
   it('range value results in VALUE error', () => {
     const engine = HyperFormula.buildFromArray([
-      ['1', '=10 / A1:A3'],
-      ['9', '=A1:A3 / 10'],
+      ['1'],
+      ['9'],
       ['3'],
+      ['=10 / A1:A3'],
```

Comment:
No, since those tests are in {arrays: false} mode. In {arrays: true} mode, we will compute this exactly as XL, and XL is by default in {arrays: true} mode. We should add the info about the flag to the docs somewhere.

Reply:
Should we add the `array: false` to those tests then to make it obvious and have those tests sealed in case we change `array` to default `true`?